test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ClusteredWorkbenchKieServerDatabasePersistentScenarioIntegrationTest.java

```diff
@@ -66,11 +66,10 @@ public static void initializeDeployment() {
             deploymentScenario = deploymentScenarioFactory
                     .getClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder()
                     .withInternalMavenRepo()
-                    .withInternalMavenRepo()
-                        .withGitSettings(GitSettings.fromProperties()
-                                                     .withRepository(REPOSITORY_NAME,
-                                                                     ClusteredWorkbenchKieServerDatabasePersistentScenario.class.getResource(
-                                                                                                                           PROJECT_SOURCE_FOLDER + "/" + Kjar.HELLO_RULES.getArtifactName()).getFile()))
+                    .withGitSettings(GitSettings.fromProperties()
+                                                .withRepository(REPOSITORY_NAME,
+                                                                ClusteredWorkbenchKieServerDatabasePersistentScenario.class.getResource(
+                                                                                                                      PROJECT_SOURCE_FOLDER + "/" + Kjar.HELLO_RULES.getArtifactName()).getFile()))
```

Comment:
I fixed the format and also removed the duplicated line "withInternalMavenRepo()".